toolchain/sem_ir/ids.h

```diff
@@ -305,6 +305,32 @@ struct BoolValue : public IdBase, public Printable<BoolValue> {
 constexpr BoolValue BoolValue::False = BoolValue(0);
 constexpr BoolValue BoolValue::True = BoolValue(1);
 
+// An integer kind value -- either "signed" or "unsigned".
+//
+// This might eventually capture any other properties of an integer type that
+// affect its semantics, such as overflow behavior.
+struct IntKind : public IdBase, public Printable<IntKind> {
+  static const IntKind Unsigned;
+  static const IntKind Signed;
+
+  // Returns whether this type is signed.
+  constexpr auto is_signed() -> bool { return *this == Signed; }
+
+  using IdBase::IdBase;
```

Comment:
```suggestion
  using IdBase::IdBase;

    // Returns whether this type is signed.
  constexpr auto is_signed() -> bool { return *this == Signed; }

```

Since IdBase is a constructor...